### PR TITLE
add symlink for all assets by default

### PR DESCRIPTION
**Pull request by xfbao1986**
add all symlink for assets refer by `manifest.json` when `config.asset_symlink = :all`

### CODE REVIEW

File: lib/asset_symlink.rb [hunk 1]
```diff
@@ -27,16 +27,25 @@ def self.execute config
...
     when NilClass
-      {}
+      result = {}
+      begin
+        manifest = JSON.parse(File.read(Rails.application.config.assets.manifest))
+        manifest["files"].each do |digested_name, value|
+          result[value['logical_path']] = value['logical_path']
+        end
+        result
+      rescue
```
Comment (fcheung) [inline lib/asset_symlink.rb]: What’s the rescue for? Not a fan of unspecified rescues like that.

File: lib/asset_symlink.rb [hunk 2]
```diff
@@ -27,16 +27,25 @@ def self.execute config
...
+    when Hash
       config
     when String
       {config => config}
-    when Array 
+    when Array
       config.inject({}) do |result, element|
         result.merge(normalize_configuration(element))
       end
     when NilClass
```
Comment (fcheung) [inline lib/asset_symlink.rb]: Maybe more explicit if this is triggered by doing something like `config.asset_symlink = :all ` ?

**Comment by xfbao1986** [comment]
@fcheung please review again

**Comment by fcheung** [comment]
Thanks!

**Pull request merged**